jobs/pipelines/powervm/zstreams/zstream-trigger-job/Jenkinsfile

```diff
@@ -0,0 +1,186 @@
+pipeline {
+    agent {
+        docker {
+            image 'quay.io/powercloud/inbound-agent:4.6-1.3'
+            args '-v /etc/resolv.conf:/etc/resolv.conf -v /var/run/docker.sock:/var/run/docker.sock'
```

Comment:
@varad-ahirwadkar   you can remove ` -v /var/run/docker.sock:/var/run/docker.sock` as this file is not using docker

Reply:
ok I will remove it